docs/sources/administration/roles-and-permissions/access-control/rbac-terraform-provisioning/index.md

```diff
@@ -0,0 +1,151 @@
+---
+aliases:
+  - /docs/grafana/latest/enterprise/access-control/rbac-provisioning/rbac-terraform-provisioning
+description: Learn about RBAC Terraform provisioning and view an example of provisioning configuration
+  for Grafana roles and role assignments.
+menuTitle: Provisioning RBAC with Terraform
+title: Provisioning RBAC with Terraform
+weight: 60
+---
+
+# Terraform RBAC provisioning
```

Comment:
nit: would be great to align RBAC with verbs in the begining of the titles

I feel like "Provising RBAC with Terraform" is a great title.

So maybe the header should be the same?